torch/utils/cpp_extension.py

```diff
@@ -271,7 +282,7 @@ def check_compiler_ok_for_platform(compiler: str) -> bool:
     return False
 
 
-def check_compiler_abi_compatibility(compiler) -> bool:
+def check_compiler_abi_compatibility(compiler) -> Tuple[bool, Tuple[str]]:
```

Comment:
Please document new returned values in `Returns` section of the comment. Also, perhaps it worth renaming the function  as its no longer a predicate, is it?
Also, as you return multiple strings in a tuple, rather than one, proper annotation should be `Tuple[str, str, str]` isn't it?